### ISSUE TITLE
api-18 inconsistency

### ISSUE THREAD

**Issue by nohal** [leamas/opencpn-libs#9]
`ocpn::api_wx32` is defined in `api-18` instead of the former `ocpn::api`, but `plugingl` always links `ocpn::api` and as such is unusable now.

Is there any real reason for this name change at all? What breaks if we simply use the old name even in `api-18`?

**Comment by leamas** [comment]
> Is there any real reason for this name change at all?

Nothing I can remember...

> What breaks if we simply use the old name even in api-18?

 the only way is to make a try, I guess. Perhaps then without removing ocpn::api_wx32 at this point close to the release i. e.:
 ```
index 3a2c073..e16d491 100644
--- a/api-18/CMakeLists.txt
+++ b/api-18/CMakeLists.txt
@@ -13,6 +13,7 @@ set(API_OCPN_MIN_VERSION "5.7.0" PARENT_SCOPE)
 
 add_library(OCPN_API_WX32 INTERFACE)
 add_library(ocpn::api_wx32 ALIAS OCPN_API_WX32)
+add_library(ocpn::api ALIAS OCPN_API_WX32)
 target_include_directories(
   OCPN_API_WX32
     INTERFACE "${CMAKE_CURRENT_SOURCE_DIR}"
```

**Comment by nohal** [comment]
Yes, this is exactly what I did locally...

**Comment by nohal** [comment]
At least in the shipdriver template there is some [voodoo using the new name](https://github.com/Rasbats/shipdriver_pi/blob/master/CMakeLists.txt#L41-L45), based on the name of the package, but it did not make me any smarter...

**Comment by leamas** [comment]
Pushed a fix to devel branch for now, will sleep and make some tests before merging to main.

**Comment by leamas** [comment]
That voodoo seems like silly me working around the core issue that the target is different on 18 compared 16.  With patch above we should be able to avoid it.


Actually, good catch!

**Comment by leamas** [comment]
... rebased devel on main, no changes

**Comment by leamas** [comment]
Fixed in d778f76. Closing
